client/components/domains/use-your-domain-step/index.jsx

```diff
@@ -355,6 +367,7 @@ export default connect(
 	state => ( {
 		currentUser: getCurrentUser( state ),
 		currencyCode: getCurrentUserCurrencyCode( state ),
+		domainsWithPlansOnly: currentUserHasFlag( state, DOMAINS_WITH_PLANS_ONLY ),
```

Comment:
@delputnam fun thing, this kind of never worked w/o the flag here. :stuck_out_tongue: